pymc3/sampling.py

```diff
@@ -16,7 +16,7 @@
 import sys
 sys.setrecursionlimit(10000)
 
-__all__ = ['sample', 'iter_sample', 'sample_ppc', 'init_nuts']
+__all__ = ['assign_step_methods', 'sample', 'iter_sample', 'sample_ppc', 'init_nuts']
```

Comment:
Yes because iter_sample needs a step function, so if you don't have one already...